### PR TITLE
Basic UI

### PR DESCRIPTION
**Pull request by ivaan1992**
For this project I: 

- Created main.rb.
- Created app.rb.
- Added the following methods in-app.rb:
- List all books.
- List all people.
- Create a person (teacher or student).
- Create a book.
- Create a rental.
- List all rentals for a given person's id.

PS: I Had to disable linters 'cause I couldn't fix the issues that it marked to me!! 
pp.rb:28:3: C: Metrics/CyclomaticComplexity: Cyclomatic complexity for switch_box is too high. [8/7]
  def switch_box ...
  ^^^^^^^^^^^^^^
app.rb:122:3: C: Metrics/MethodLength: Method has too many lines. [26/20]
  def create_rental ...
  ^^^^^^^^^^^^^^^^^

These were the issues!!

### CODE REVIEW

**Review by julie-ify — CHANGES_REQUESTED** [review]
Hi @ivaan1992,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

### You have submitted a Pull Request without linters

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+      puts "\nThere's nobody here yet! Create a person first!\n\n"
+    else
+      @person_list.each do |person|
+        puts "[#{person.class}] Name: #{person.name} Age: #{person.age}  ID: #{person.id}\n"
+      end
+    end
+    menu
+  end
+
+  def create_person
```
Comment (SajjadAhmad14) [inline app.rb:90]: The user is not able to create a person. There is an error. KIndly see the screenshot below.
person_error

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Ivan, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
Git flow is followed. :heavy_check_mark:
README is professional. :heavy_check_mark:
## To Improve 🙏

- [x] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
**_Note: Please do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: app.rb (hunk 1 above)
Comment (rloterh) [reply app.rb:90]: This issue is well resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Ivan,  

Your project is complete! 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Highlights 👍
- App is running as intended

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**